src/components/ec-table/ec-table.vue

```diff
@@ -119,7 +123,10 @@ export default {
       this.$emit('sort', columnName);
     },
     getColumnWidth(column) {
-      return column && column.minWidth ? { minWidth: column.minWidth } : null;
+      if (column && (column.maxWidth || column.minWidth)) {
+        return { maxWidth: column.maxWidth, minWidth: column.minWidth };
```

Comment:
@jafcalvente Don't we have any problem if we pass maxWidth but no minWidth being minWidth undefined?

Reply:
We should not, because when using the Object Syntax the defined styles are only applied if the condition is true.

Reply:
https://github.com/vuejs/vue/blob/aef5b4e47811cb842315bd27d6919650da45279b/src/platforms/web/server/modules/style.js#L23

The value must be string or number, otherwise it's ignored. So minWidth set to null or undefined is just not rendered.